utils/installer/install.sh

```diff
@@ -110,13 +110,12 @@ installpacker() {
 }
 
 cloneconfig() {
+	declare USER_BIN_DIR="/usr/local/bin"
```

Comment:
please move this to line #1 of the file 😅

Reply:
I moved it to line 6

Reply:
@abzcoding I moved it into line 6 
Should I move it to 2 or 4?

Reply:
line 4 would be much appreciated 🥺